stories/spectra/nasqad.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import { Meta } from '@storybook/react';
+import React, { useState, useEffect } from 'react';
+
+import { Axis, LineSeries, Plot, Heading } from '../../src';
+import type { Series } from '../../src/types';
+import srcData from '../data/nasdaq.json';
+
+export default {
+  title: 'Experimental spectra/Nasqad',
```

Comment:
Please check everywhere so that it is Nasdaq and not Nasqad

Reply:
Also please move this to the "Examples" category. This data is not an experimental spectrum